apps/email/js/cards/message_list.js

```diff
@@ -681,7 +681,13 @@ return [
       // date
       var dateTime = message.date.valueOf();
       dateNode.dataset.time = dateTime;
-      dateNode.textContent = dateTime ? date.prettyDate(message.date) : '';
+      if (dateTime) {
+        date.relativeDate(message.date).then(string => {
+          dateNode.textContent = string;
+        });
+      } else {
+        dateNode.textContent = '';
+      }
       // subject
```

Comment:
Is there a synchronous option for data.relativeDate? This async version does not work well for message_list or message_list_search since that work is done while updating elements in our virtual scroll list. Fast scrolling means that the async return will not target the right nodes. I was able to reproduce the issue on device.

If the issue is needing to wait for ready() inside the l10n library, email's l10n.js waits for .once() already, we could look at modifying that more if we need to make sure some async one time setup needs to happen in the l10n/moz_intl infrastructure before calling these methods.